signalfx/signalflow/__init__.py

```diff
@@ -33,8 +33,8 @@ def _get_params(self, **kwargs):
 
     def execute(self, program, start=None, stop=None, resolution=None,
                 max_delay=None, persistent=False, immediate=False,
-                disable_all_metric_publishes=None, withDerivedMetadata=False,
-                resolutionAdjustable=False):
+                disable_all_metric_publishes=None, withDerivedMetadata=None,
+                resolutionAdjustable=None):
```

Comment:
@kchengsf you have the most context about these flags. Aren't these supposed to only be used internally ? I'm worried about exposing those in the SDK especially if they aren't documented.